### ISSUE TITLE
BugReport: MouseMovement in VST3 in Plugin Host

### ISSUE THREAD

**Issue by Yoko90** [julianstorer/JUCE#93]
having some trouble with the mosueMove (...) in VST3 in Plugin Host, it gets only called on a mouse down event not on a mouse move.

To reproduce, make a new Plugin with the Projucer and add this to the Editor and Debug with Plugin Host in VST3

```
public:
   void mouseMove (const MouseEvent& e) override
    {
        DBG ("mouse moved");
    }
```
The weird thing is, it works with the Demo Plugin project, but not with a new generated one.

I'm on latest develop with a freshly built Projucer and freshly built Plugin Host, Mac OS X 10.11.4, XCode 7.3.1

**Comment by tpoole** [comment]
Hi Yoko90,

There's an option in the Projucer - "Plugin editor requires keyboard focus". This is enabled for the Demo Plugin project but disabled by default. Does selecting this option fix the problem for you?

**Comment by tpoole** [comment]
This is fixed in 08b2116c.
